src/routes/gmr-results.js

```diff
@@ -0,0 +1,31 @@
+import { paths } from './route-constants.js'
+import { results } from './results.js'
+import { getRelatedImportDeclarations } from '../services/related-import-declarations.js'
+import { mapVehicleDetails } from '../models/vehicle-details.js'
+import { mapGmrCustomsDeclarations } from '../models/customs-declarations.js'
+
+export const gmrResults = results(true, paths.GMR_RESULTS, async (request, h) => {
+  const searchTerm = request.query.searchTerm.trim()
```

Comment:
change to 
`const searchTerm = request.query[queryStringParams.SEARCH_TERM].trim()